R/as_tibble.R

```diff
@@ -2,6 +2,7 @@
 #'
 #' @description
 #' \lifecycle{maturing}
+#' \lifecycle{superseded}
```

Comment:
I wouldn't put the badge up here if you're just superseding part of the capabilities